### DISCUSSION TITLE
Not really a bug. L@@king for migration help v 1-2 and webpack/react/saas.

### DISCUSSION THREAD

**Discussion by mesh-newbie** [yarnpkg/berry#3101] [Q&A]
Hi Guys, <info for newbies like me>
Wanting to update/migrate Y 1 -> 2 (as it looks/works great 👍🏻)), but doing it on a large older project/repo is proving difficult/impossible because of CRA.

**What works: (@imports from the node_folders)**
`@import "animate-sass/helpers/mixins.scss"; // was @import "../../../../../node_modules/animate-sass/helpers/mixins.scss"`

**What doesn't work anymore: (sass/css relative urls - 'yarn start')**
`p {
    background-image: url('../assets/img/mycoolmap.jpg');
}`
_Module not found: You attempted to import ../assets/img/mycoolmap.jpg which falls outside of the project src/ directory. Relative imports outside of src/ are not supported._

Given that the project is updated by multiple devs, I'm trying to cater for both Y v1 and 2 users (though this might not be possible?). I know this is **not** a Y problem.. just looking for possible solutions to a project that was CRA in early 2019.
Have considered the following (but not practical) : ( CRA see [issue here](https://github.com/facebook/create-react-app/issues/9937) )

- Upgrade CRA and eject
- Move the /public folder into /src
- Craco
- Downgrading react-scripts

Have exhausted a week on finding answers without success. All alternative suggestions most welcomed 😀

Cheers.

**Comment by arcanis** [comment]
Unfortunately it's difficult to make Y1 and Y2 play together on the same repository, since they use different configuration/lockfile formats. However you should be able to migrate painlessly using the `node_modules` linker (cf the [step by step](https://yarnpkg.com/getting-started/migration#step-by-step)).

**Reply by mesh-newbie** [reply]
Cheers @arcanis.
Acknowledged re only a single version of Yarn. 👍🏻
I was unable to find a solution (or even a workaround) with node_modules linker and the step-by-step guide mentioned. ☹️
